src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/GenerateDocumentHandler.java

```diff
@@ -208,7 +208,7 @@ private String getFtpaApplicationDeadline(AsylumCase asylumCase, LocalDate appea
         boolean isAda = asylumCase.read(IS_ACCELERATED_DETAINED_APPEAL, YesOrNo.class).orElse(YesOrNo.NO).equals(YesOrNo.YES);
 
         if (isInternalCase) {
-            return getFtpaApplicationDeadlineForInternalCase(appealDate, isAda).format(DateTimeFormatter.ofPattern("d MMMM yyyy"));
+            return getFtpaApplicationDeadlineForInternalCase(appealDate, isAda).format(DateTimeFormatter.ofPattern("d MMM yyyy"));
```

Comment:
Should we be formatting the date here? I think we should return the LocalDate.toString() value from getFtpaApplicationDeadlineForInternalCase

Reply:
Updated LocalDate toString instead of using date formatter